assets/css/main.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('@/assets/fonts/Roboto-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+.titletext {
+    font-family: Roboto;
+    font-size: 30px;
+    font-weight: 700;
+    line-height: 35px;
+    letter-spacing: 0em;
+    text-align: left;
+}
+
+.cardDescription {
+    font-family: Roboto;
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 28px;
+    letter-spacing: 0em;
+    text-align: left;
+
+}
+
+.caption {
+    font-family: Roboto;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 0em;
+    text-align: left;
+    color: #BBBDD0;
+}
+
+.infoText {
+    font-family: Roboto;
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 28px;
+    letter-spacing: 0em;
+    text-align: center;
+}
+
+.card {
+    width: 100%;
+    padding: 16px;
+    border-radius: 5px;
+    border: 1px solid #DDE2FF;
+    background-color: white;
+}
+
+.spaceBetween {
+    display: flex;
+    justify-content: space-between;
+}
```

Comment:
`.spaceBetween` should only hold `justify-content: space-between`. 
You can also do the same for flex. Also if you wish to use some atomic classes I would suggest you to make the names more smaller and use kebab casing.

Reply:
Resolved